src/Elastic.Apm/AgentComponents.cs

```diff
@@ -72,6 +72,13 @@ public class AgentComponents : IApmAgent, IDisposable
 				}
 				else
 					Logger.Info()?.Log("The Elastic APM .NET Agent is disabled - the agent won't capture traces and metrics.");
+
+				TracerInternal = new Tracer(Logger, Service, PayloadSender, ConfigurationStore,
+					currentExecutionSegmentsContainer ?? new CurrentExecutionSegmentsContainer(), ApmServerInfo, breakdownMetricsProvider);
+
+#if NET5_0
+				new OpenTelemetry.ElasticActivityListener(this, TracerInternal);
```

Comment:
Since the activity listener is enabled by default, should there be a way to opt-out e.g. through configuration?

Reply:
Good point. I added a config for this.

For the beta phase, I'd make it opt-in. By default it's disabled and people can enable it. Once it's GA I'd flip it to enabled by default.